.rubocop.yml

```diff
@@ -82,3 +82,7 @@ Style/PreferredHashMethods:
 
 Minitest/AssertEmptyLiteral:
   Enabled: true
+
+Minitest/AssertNil:
+  Enabled: true
```

Comment:
If it was previously in the TODO file that implies it was already enabled and this is redundant.